deltachat-ios/Controller/NewChatViewController.swift

```diff
@@ -135,7 +134,7 @@ class NewChatViewController: UITableViewController {
 
     override func tableView(_: UITableView, numberOfRowsInSection section: Int) -> Int {
         if section == sectionNew {
-            return 3
+            return 2
```

Comment:
can we please have a sectioNewRowCount at line 11 for that?

Reply:
sure